examples/cli-options-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Where is this used?

Reply:
It's just an example of how option config can be used. Not used anywhere.